docs/education/dlt-fundamentals-course/lesson_2_dlt_sources_and_resources_create_first_dlt_pipeline.py

```diff
@@ -0,0 +1,719 @@
+    )
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        r"""![Lesson_2_dlt_sources_and_resources_Create_first_dlt_pipeline_img1.png](https://storage.googleapis.com/dlt-blog-images/dlt-fundamentals-course/Lesson_2_dlt_sources_and_resources_Create_first_dlt_pipeline_img1.png)"""
+    )
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""
+    ## Recap of [Lesson 1](https://github.com/dlt-hub/dlt/blob/master/docs/education/dlt-fundamentals-course/lesson_1_quick_start.ipynb) 👩‍💻🚀
+    1. Created a pipeline, loaded toy data into DuckDB, and viewed the load info.
+    2. Used the `dlt.pipeline` and `pipeline.run` methods.
+    3. Queried data and viewed tables with DuckDB, the `sql_client`, and the dlt `dataset`.
+
+    Now, let's move on to the next lesson to learn more! 🚀
+    """)
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""
+    **Here, you will learn how to:**
+    - Run a simple pipeline with different types of data, such as dataframes, databases and REST APIs.
+    - Use `dlt.resource`, `dlt.source` and `dlt.transformer`.
+    - Build your first dlt pipeline for a REST API.
+    """)
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""
+    ---
+    ## **`dlt` resources**
+    """)
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""
+    ---
+    ### List of dicts
+
+
+    In the previous lesson, we simply used a list of dictionaries that essentially represents the `pokemon` table.
+    """)
+    return
+
+
+@app.cell
+def _():
+    import dlt
+    data = [{'id': '1', 'name': 'bulbasaur', 'size': {'weight': 6.9, 'height': 0.7}}, {'id': '4', 'name': 'charmander', 'size': {'weight': 8.5, 'height': 0.6}}, {'id': '25', 'name': 'pikachu', 'size': {'weight': 6, 'height': 0.4}}]
+    # Sample data containing pokemon details
+    pipeline = dlt.pipeline(pipeline_name='quick_start', destination='duckdb', dataset_name='mydata')
+    _load_info = pipeline.run(data, table_name='pokemon')
+    # Set pipeline name, destination, and dataset name
+    # Run the pipeline with data and table name
+    print(_load_info)
+    return data, dlt, pipeline
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        r"""A better way is to wrap it in the `@dlt.resource` decorator which denotes a logical grouping of data within a data source, typically holding data of similar structure and origin:"""
+    )
+    return
+
+
+@app.cell
+def _(data, dlt):
+    from dlt.common.typing import TDataItems, TDataItem
+
+    @dlt.resource(table_name='pokemon_new')
+    def my_dict_list() -> TDataItems:
+    # Create a dlt resource from the data
+        yield data  # <--- we set new table name
+    return TDataItem, TDataItems, my_dict_list
```

Comment:
The comment is referring to the decorator
```suggestion
    @dlt.resource(table_name='pokemon_new')  # <--- we set new table name
    def my_dict_list() -> TDataItems:
    # Create a dlt resource from the data
        yield data
    return TDataItem, TDataItems, my_dict_list
```